src/app/offers/offer-details/offer-details.component.html

```diff
@@ -230,25 +228,71 @@ <h1 class="app-basic-title">{{ states.project.name }}</h1>
                 </div>
             </div>
 
-            <div class="left__news" *ngIf="states.news != null">
-                <div class="news__title">
-                    Project news
+            <div class="right__news" *ngIf="states.project.news.length !== 0">
+                <div class="news__title app-page-subtitle">
+                    {{ 'offers.offer_details.news.subtitle' | translate }}
                 </div>
-                <div *ngFor="let news of states.news" class="news__item">
-                    <div class="news__wrapper">
-                        <a href="{{ news.url }}" target="_blank" rel="noopener">{{ news.url }}</a>
-                    </div>
+
+                <div *ngFor="let news of states.news">
+                    <ng-container *ngIf="news.open_graph !== null else newsPreviewFallback">
+                        <div class="news__item-preview">
+                            <div class="preview__inner">
+                                <div class="preview__img">
+                                    <img class="w-100" src="{{ news.open_graph.image.url }}"
+                                         onerror="this.onerror=null; this.src='../../../../assets/noimage.png'">
+                                </div>
+                                <div class="preview__content">
+                                    <div class="preview__title">
+                                        {{ news.open_graph.title === null ?
```

Comment:
you can just set `!news.open_graph.title` because we don't mind if it's `undefined`, `null`, or an empty string.

Reply:
Right, awesome. Will change it now